### PR TITLE
mgmt, update pom for javadoc for Lite

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
Ignore the formatting, as new release will generate new pom anyway.

File: sdk/vmwarecloudsimple/azure-resourcemanager-vmwarecloudsimple/pom.xml [hunk 1]
```diff
@@ -38,7 +38,10 @@
     </developers>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+        <jacoco.skip>true</jacoco.skip>
+    <codesnippet.skip>false</codesnippet.skip>
```
Comment (haolingdong-msft) [inline sdk/vmwarecloudsimple/azure-resourcemanager-vmwarecloudsimple/pom.xml:42]: indent

**Review by haolingdong-msft — APPROVED** [review]

**Pull request merged**